index.js

```diff
@@ -31,8 +31,12 @@ function init() {
   ui.setCustomDatasetCallback(csv => selectDataset(data.parseCsv(csv)))
   ui.setHightlightCallback(id => canvas.getGrid().onHighlightGeo(id))
   ui.setUnhighlightCallback(() => canvas.getGrid().resetHighlightedGeo())
-  ui.setResolutionChangedCallback(value => {
-    hexagonGrid.setTileEdge(value)
+  ui.setResolutionChangedCallback((metricPerTile, sumMetric) => {
```

Comment:
hmm I sort of worry about these anonymous callback functions being overly complex inline like this? would it make sense to break it out as it's own named function?